jekyll/_cci2/insights-partnerships.md

```diff
@@ -13,7 +13,43 @@ version:
 
 {:toc}
 
-This document describes how you can connect Insights data with third party providers. Currently we support integrations with [Datadog](https://www.datadoghq.com/) and [Sumo Logic](https://www.sumologic.com/).
+This document describes how you can connect Insights data with third party providers. Currently we support integrations with [New Relic](https://newrelic.com/), [Datadog](https://www.datadoghq.com/) and [Sumo Logic](https://www.sumologic.com/).
+
+## New Relic integration
+
+The [New Relic CircleCI integration](https://newrelic.com/instant-observability/circleci) allows users to view analytical data about their CircleCI jobs within the New Relic monitoring stack.
+
+Complete these steps to set up a CircleCI webhook to forward your logs to New Relic:
+
+**Step 1.** Log in to [CircleCI](https://app.circleci.com/projects).
+
+**Step 2.** Go to one of your CircleCI projects.
+
+**Step 3.** Click **Project settings**.
+
+**Step 4.** In the sidebar of your Project Settings, click **Webhooks**.
+
+**Step 5.** Click **Add webhook**.
+
+**Step 6.** Name your webhook.
+
+**Step 7.** Enter New Relic's **Logs endpoint:**
+
+US:
```

Comment:
Is this US and Canada?

Reply:
@corinnesollows, While creating a new account at New Relic, a user can choose their preferred data storage region i.e., US or Europe. Please note that the data center chosen doesn’t have to match with the physical location of the user.

Reply:
@corinnesollows This sounds good to me, do we need any further changes or are we OK to go ahead and merge?

Reply:
@kbatuigas, everything is good, please go ahead! 
Thank you.